src/main/scala/draughts/Pos.scala

```diff
@@ -44,6 +45,16 @@ sealed case class Pos100 private (x: Int, y: Int) extends PosMotion(5 * (y - 1)
   lazy val moveLeft  = movesHorizontal.get(fieldNumber).map(_(0)).filter(_ > 0) flatMap posAt
   lazy val moveRight = movesHorizontal.get(fieldNumber).map(_(1)).filter(_ > 0) flatMap posAt
 
+  def getDistFromSide(): Int = {
```

Comment:
A couple of stylistic things:

Because you have no parameters the brackets are unnecessary and just mean that to call this function you have to do `getDistFromSide()` whereas if you removed them from the function definition (here) you can also remove them from anywhere that calls this function.
You also don't need the outer `{`/`}` because you have a singular statement in the function.